packages/common/src/utils/validationUtils.test.ts

```diff
@@ -1152,8 +1152,6 @@ describe('Set initial values to a certificate', () => {
 
       expect(certificate.data['1.1'].readOnly).toBe(true)
       expect(certificate.data['1.2'].readOnly).toBe(true)
-      expect(certificate.data['1.1'].visible).toBe(true)
```

Comment:
Instead of removing the visibility check, would it be good to keep makeing that control and instead remove the visibility = false in function clearValues() that gets called in the start of each test? Or change it so it sets visible = undefined

Reply:
ok, i will change it to === undefined